pkgs/tools/typesetting/soupault/default.nix

```diff
@@ -22,9 +23,17 @@ ocamlPackages.buildDunePackage {
     owner = "PataphysicalSociety";
     repo = pname;
     rev = version;
-    sha256 = "sha256-M4gaPxBxQ1Bk2C3BwvobYHyaWKIZgQ6buZ6S5wBlvPg=";
+    sha256 = "MblwVacfK9CfoO0TEND+bqdi7iQayBOJKKOhzE7oiVk=";
   };
 
+  patches = lib.lists.optional
+    (lib.strings.versionAtLeast ocamlPackages.camomile.version "2.0.0")
+    (fetchpatch {
```

Comment:
Please make this unconditional if possible

Reply:
What does this mean & what is the problem? There was already a long conversation for the 4.5.0 merge request (not merged because...) because Camomile 2.0 is not stable enough for Nixpkgs. This patch, written by the maintainer, allows `soupault` to upgrade, while the OCaml community works to make Camomile more stable/backwards compatible. If/when a Camomile 2.x upgrade _is_ merged, we don’t want Soupault 4.6.0 to use the 1.x patch at all, but instead use the actual 2.x version.

Reply:
(though I was using the wrong switch, needed `lib.strings.versionOlder`)

Reply:
Or just replace it with `versionOlder`.

Reply:
> What does this mean & what is the problem?

the conditional should be removed if possible because when updating it might be forgotten and makes the patch situation in general more complex